docs/development/local-development.md

```diff
@@ -46,11 +52,11 @@ The following steps work to set up a brand new Windows 10 installation for devel
 - In an Administrator PowerShell prompt, run `npm install -global npm` and then `npm --add-python-to-path='true' --debug install --global windows-build-tools`
 - Install [Yarn](https://yarnpkg.com/lang/en/docs/install/#windows-stable)
 - Install `mock` for Python:
-
   - `curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py`
   - `python get-pip.py`
   - `rm get-pip.py`
   - `python -m pip install mock`
+- Install [Java](https://docs.oracle.com/en/java/javase/12/install/installation-jdk-microsoft-windows-platforms.html)
```

Comment:
Personally I would just link to https://adoptopenjdk.net/. Feel free to ignore this comment

Reply:
Good idea! I changed it.